CONTRIBUTING.md

```diff
@@ -1,7 +1,24 @@
 # Contribution Guidelines for the Firefox Accounts Content Server
 
-Anyone is welcome to help with Firefox Accounts. Feel free to get in touch with other community members on Matrix, the
-mailing list or through issues here on GitHub.
+Anyone is welcome to help with Firefox Accounts, but it's important to
+recognize that Firefox Accounts is a complex tool written by Mozilla and used
+solely within Mozilla or by parties tightly coupled to Mozilla.  The teams goal
+is focused on serving Mozilla's requirements -- not being easy to set up or
+easily usable in other scenarios.
+
+In order to scale effectively, we may use technolgies which have significant
+barriers to entry (eg. cost or expertise).
+
+If you're filing bugs, working on documentation, or helping reproduce a
+problem, thank you for your help and please read below for guidelines.
+
+If you want to write code, and you are not an experienced programmer you will
+probably be more successful [looking for other projects at
+Mozilla](https://whatcanidoformozilla.org) to contribute to.  The Firefox
+Accounts team is happy to support open source contributions but we have limited
+time to assist in getting the FxA codebase up and running on other platforms.
```

Comment:
Did we want to add something about `help wanted` or `good first issue` labels here?

Overall, I think this is a good addition to this doc. I do think if folks can get the stack running, the ones we label as `good first issue` are pretty straightforward. I think another hurdle for contributors trying to grab anything that's not labeled as such is we oftentimes file issues with the expectation that one of us will grab it - there's an assumed level of knowledge/familiarity, and a contributor might need additional context to understand what some issues are even about.

I also don't want to discourage anyone from trying, though. Maybe the way you've phrased it here is the right way to convey this.

Reply:
+1, I do think it's important to restrict outside contributors to some specific good-first-bug or similar label. We want to avoid putting contributors under sprint-related time pressure, and also avoid staff spending a lot of time getting a contributor up to speed, only to have them disappear. An effective way to avoid both problems is to point contributors at lower-priority bugs with a low learning curve.

Reply:
I added a sentence.  Thanks